### ISSUE TITLE
Fix build validation warnings

### ISSUE THREAD

**Issue by zalun** [mozilla/platform-status#373]
```
Validation warnings:
        background-sync.md: missing required property "mdn_url"
        ....
        webgl-2.md: missing caniuse_ref
```

**Comment by marco-c** [comment]
Most of the warnings can't be fixed, because what's missing actually doesn't exist (for example, there isn't a MDN page for the Background Sync API).
It's useful to keep the warnings anyway because we can periodically check if we can fix some of them.

I've fixed one with #410.

**Comment by marco-c** [comment]
Also fixed another with #411.
